DNN Platform/Library/Services/Log/EventLog/LogInfo.cs

```diff
@@ -30,37 +32,64 @@ public LogInfo(string content)
             : this()
         {
             this.Deserialize(content);
-        }
-
-        public string LogGUID { get; set; }
-
-        public string LogFileID { get; set; }
-
+        }
+
+        /// <inheritdoc />
+        public string LogGuid { get; set; }
```

Comment:
These renamed properties need to use explicit interface implementation so that we're not exposing multiple members that only differ by case (for VB.NET support)
```suggestion
        string ILogInfo.LogGuid { get; set; }
```

Reply:
Is it still okay to deprecate the old ones?

Reply:
This has been updated in the latest code push, thanks for the feedback